docs/narr/urldispatch.rst

```diff
@@ -1100,52 +1097,73 @@ was matched (see :class:`pyramid.interfaces.IRoute` for the API of such a route
 object).
 
 ``info['match']`` is useful when predicates need access to the route match.
-For example:
+Imagine you want to define a route when a part of the URL matches some
+specific values.  You could define three view configurations, each one
+with its own ``match_param`` value (see :ref:`predicate_view_args`), but
+you want to think of it as one route, and associate it with one view.
+See this code:
 
 .. code-block:: python
     :linenos:
 
-    def any_of(segment_name, *allowed):
-        def predicate(info, request):
-            if info['match'][segment_name] in allowed:
-                return True
-        return predicate
+    class AnyOfPredicate:
```

Comment:
Note that these example classes do not redundantly inherit from `object`, even though I saw that many parts of the current docs still have that.

Reply:
It's fine to drop the object parent, I'd accept a PR that did it consistently to the whole codebase but it's unimportant right now - new stuff is free to not use it like in this PR.